jvb/src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -1118,22 +1118,8 @@ public JSONObject getDebugState()
      * @param enabled the state of the feature.
      */
     public abstract void setFeature(EndpointDebugFeatures feature, boolean enabled);
-    
-    public abstract boolean isFeatureEnabled(EndpointDebugFeatures feature);
-
-    @Override
-    public boolean isSendingAudio()
-    {
-        // The endpoint is sending audio if we (the transceiver) are receiving audio.
-        return transceiver.isReceivingAudio();
-    }
 
-    @Override
-    public boolean isSendingVideo()
-    {
-        // The endpoint is sending video if we (the transceiver) are receiving video.
```

Comment:
I find the comment useful to remind of the perspective

Reply:
Yeah I wondered about those, too.  Brought them back.